src/SnackbarItem/SnackbarItem.tsx

```diff
@@ -194,6 +204,10 @@ const SnackbarItem: React.FC<SnackbarItemProps> = ({ classes, ...props }) => {
             timeout={175}
             in={collapsed}
             onExited={callbacks.onExited}
+            classes={{
+                wrapper: clsx(classes.collapseWrapper, { [classes.collapseWrapperDense]: dense }),
+                container: classes.collapseContainer
```

Comment:
FYI with @material-ui/core 4.12.1 this wants to be

```tsx
    root: classes.collapseContainer
```

rather than `container`.